example.tex

```diff
@@ -56,11 +56,9 @@ \subsubsection{Even more fun!}
 
 \lipsum[3]
 
-\begin{figure}[!t]
-	\begin{paperbox}{Behold, the Paperbox!}
-		The \lstinline!paperbox! is used as a sidebar. It does not break over columns and is best used with a figure environment to float it to one corner of the page where the surrounding text can then flow around it.
-	\end{paperbox}
-\end{figure}
+\begin{paperbox}[float, floatplacement=!t]{Behold, the Paperbox!}
```

Comment:
Could you change this to `float=!t`? `floatplacement affects the default placement for `float`.

Reply:
Done!